### PR TITLE
Minor fixes.

### PR DESCRIPTION
**Pull request by JakeDaBoss**

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -85,15 +85,15 @@
...
-		/obj/item/weapon/storage/box/flashbangs,
+//		/obj/item/weapon/storage/box/flashbangs,
 		/obj/item/weapon/storage/box/teargas,
 		/obj/item/weapon/storage/belt/holster/security,
 		/obj/item/weapon/reagent_containers/spray/pepper,
 		/obj/item/weapon/melee/baton/loaded,
 		/obj/item/weapon/storage/box/holobadge,
 		/obj/item/device/holowarrant,
 		/obj/random/tool,
-		/obj/item/weapon/rig/security,
```
Comment (NanakoAC) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:96]: why remove the security rig from their lockers?

File: code/modules/projectiles/guns/ds13/heavy_pulse.dm [hunk 2]
```diff
@@ -51,13 +51,13 @@
 	heat = 0
 
 	//Enters overheating state when heat gets this high
-	var/max_heat = 1
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/ds13/heavy_pulse.dm]: this should remain 1, tweak the rate that heat builds up instead

File: code/modules/projectiles/guns/ds13/pulse_rifle.dm [hunk 3]
```diff
@@ -12,7 +12,7 @@ The Pulse Rifle is the standard-issue service rifle of the Earth Defense Force a
 	w_class = ITEM_SIZE_HUGE
 	handle_casings = CLEAR_CASINGS
 	magazine_type = /obj/item/ammo_magazine/pulse
-	allowed_magazines = /obj/item/ammo_magazine/pulse
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/ds13/pulse_rifle.dm:15]: does this not work on subtypes? It really should

File: html/changelogs/TheLion1675-Bugfixing.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - bugfix: "Fixed a present bug where pulse rifles would not accept HV magazines."
+  - bugfix: "Fixed an issue in piping and power which caused issues in medical isolation."
+  - bugfix: "Fixed an issue in the CSCIO's office where two consoles weren't able to be interacted with."
+  - bugfix: "Increased SMO's office to fix a similar issue as above."
+  - bugfix: "Fixed the missing taser from the CSECO's locker."
+  - bugfix: "Fixed an accidental duplication of RIG spawns in the CSECO locker."
+  - rscdel: "Removed the flash from base locker spawns. Will be re-added when they are fixed. When? I don't know, ask Nanako."
```
Comment (NanakoAC) [inline html/changelogs/TheLion1675-Bugfixing.yml]: this is a bit passive aggressive for a changelog. Don't put in lines that would expose a team member to potential harassment

**Review by NanakoAC — CHANGES_REQUESTED** [review]
the changelog thing

File: code/modules/projectiles/guns/ds13/pulse_rifle.dm (hunk 3 above)
Comment (JakeDaBoss) [reply code/modules/projectiles/guns/ds13/pulse_rifle.dm:15]: It doesn't. It's been reported a few times that HV mags cannot be inserted.

File: code/modules/projectiles/guns/ds13/heavy_pulse.dm [hunk 5]
```diff
@@ -57,19 +57,19 @@
...
 
 	//When overheating, we exit the overheat state if heat drops below this value
-	var/overheat_min = 0.5
+	var/overheat_min = 0.65
 
 	//Loses this much heat per second. Approx 250 seconds to cooldown completely
 	var/cooldown_per_second	=	0.004
 
 	//Heat gained per bullet fired
-	var/heat_per_shot = 0.005
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/ds13/heavy_pulse.dm:69]: this is a pretty crazy change, more gradual. 0.004 would do

File: code/modules/projectiles/guns/ds13/heavy_pulse.dm [hunk 6]
```diff
@@ -57,19 +57,19 @@
 	var/overheating = FALSE
 
 	//While we're overheating, we cooldown this much faster
-	var/overheat_cooldown_mult = 1.25
+	var/overheat_cooldown_mult = 1.30
 
 	//When overheating, we exit the overheat state if heat drops below this value
-	var/overheat_min = 0.5
```
Comment (NanakoAC) [inline code/modules/projectiles/guns/ds13/heavy_pulse.dm:63]: i didnt notice this before, dont touch this one, leave at half

**Review by NanakoAC — APPROVED** [review]

**Pull request merged**